static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -52,6 +62,7 @@ function GroupReplays({group}: Props) {
     location,
     organization,
   });
+  const {allMobileProj} = useAllMobileProj();
```

Comment:
oo sorry, i meant, can we move the `visibleColumns` definition to under here so that we don't need a function? (we don't need to pass in `allMobileProj` then, can just case directly on `allMobileProj`

Reply:
actually, what do you think is better🤔 because i see with that method we'd have to define the const twice

Reply:
there's three functional components using `VISIBLE_COLUMNS` in this file so I don't think we can do it without a function :(